README.rst

```diff
@@ -139,3 +122,30 @@ for more information.
 
     Random number generation can be controlled during testing by setting
     the ``SKLEARN_SEED`` environment variable.
+
+Submitting a Pull Request
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before opening a Pull Request, have a look at the
+full Contributing page to make sure your code complies
+with our guidelines: http://scikit-learn.org/stable/developers/index.html
+
+
+Project history
+---------------
+
+The project was started in 2007 by David Cournapeau as a Google Summer
+of Code project, and since then many volunteers have contributed. See
+the AUTHORS.rst file for a complete list of contributors.
+
+The project is currently maintained by a team of volunteers.
+
+**Note** `scikit-learn` was previously referred to as `scikits.learn`.
+
+
+Communication
+-------------
+
+- Mailing list: https://lists.sourceforge.net/lists/listinfo/scikit-learn-general
```

Comment:
This is actually the old mailing list. Can you please change that to https://mail.python.org/mailman/listinfo/scikit-learn ?